### DISCUSSION TITLE
Granting a permission to a project should be applied to all future subprojects & workflowitems

### DISCUSSION THREAD

**Discussion by georgimld** [openkfw/TruBudget#1541] [Ideas]
One idea we had was to implement an option that allows users to grant a permission to a project that should be applied to all future subprojects and workflowitems. Currently, when granting a permission to a project, additional actions are created that grant permissions to subprojects and workflowitems as well, however only to existing ones. Future subprojects and workflowitem do not have the specific permissions. 

- one idea would be to mark this granting of permissions with a flag that would suggest the permission should be granted for all future resources that will be created in this project (for a specific user)

Please feel free to discuss the above mentioned idea or come up with additional solutions.

**Comment by jzakotnik** [comment]
Thinking about analogies that people use to know, e.g. windows/linux folders that inherit permissions or gitlab groups where again projects inherit permissions - this sounds pretty intuitive. Following this, permissions would be inherited unless they are in purpose changed from the inherited permission, what do you think?

**Reply by jzakotnik** [reply]
Should we convert this to a enhancement issue or is there a discussion still needed?

**Comment by georgimld** [comment]
Closing this discussion, at the feature was implemented in PR #1596
